cmd/release-controller-api/http_helper.go

```diff
@@ -240,12 +240,18 @@ func canLink(tag imagev1.TagReference) bool {
 	}
 }
 
-func tableLink(config *releasecontroller.ReleaseConfig, tag imagev1.TagReference) string {
+func tableLink(config *releasecontroller.ReleaseConfig, tag imagev1.TagReference, inconsistencies releasecontroller.Inconsistencies) string {
 	if canLink(tag) {
+
 		if value, ok := tag.Annotations[releasecontroller.ReleaseAnnotationKeep]; ok {
 			return fmt.Sprintf(`<td class="text-monospace"><a title="%s" class="%s" href="/releasestream/%s/release/%s">%s <span>*</span></a></td>`, template.HTMLEscapeString(value), phaseAlert(tag), template.HTMLEscapeString(config.Name), template.HTMLEscapeString(tag.Name), template.HTMLEscapeString(tag.Name))
 		}
-		return fmt.Sprintf(`<td class="text-monospace"><a class="%s" href="/releasestream/%s/release/%s">%s</a></td>`, phaseAlert(tag), template.HTMLEscapeString(config.Name), template.HTMLEscapeString(tag.Name), template.HTMLEscapeString(tag.Name))
+		if strings.Contains(tag.Name, "night") && inconsistencies.ImageStreamTagInconsistency != nil {
+			return fmt.Sprintf(`<td class="text-monospace"><a class="%s" href="/releasestream/%s/release/%s">%s </a> <img src="warning.svg" title=%s   width="23" height="23"  /> </td>`, phaseAlert(tag), template.HTMLEscapeString(config.Name), template.HTMLEscapeString(tag.Name), template.HTMLEscapeString(tag.Name), inconsistencies.ImageStreamTagInconsistency)
```

Comment:
This is presenting funny when rendered...
1. There are weird spaces included at the end of the tag name (i.e. 4.11.0-0.nightly-2022-02-27-122819)
2. The Warning icon has a weird tooltip (the pull spec) when it should probably be something specific to the icon itself
3. The Warning icon should itself be a link to the `/releasetag/{tag}/inconsistency` page

Reply:
Removed the white space after the tag.
The tooltip was displaying the inconsistency struct, but it is not rendering properly since it is very large.
I  removed that and added a generic tooltip. 

The icon itself is now a link to /releasetag/{tag}/inconsistency
![Screenshot from 2022-03-01 18-15-04](https://user-images.githubusercontent.com/77680331/156216682-305fa919-af61-4901-a2e5-89c398a2636b.png)
consistency.